R/internals.R

```diff
@@ -46,17 +46,28 @@ spf <- function(...) stop(sprintf(...), call. = FALSE)
   return(list(result = reslist, headers = req$headers))
 }
 
+meetup_api_prefix <- function() {
```

Comment:
this is because we might want to replace this URL with the URL to a fake app for testing.

Reply:
at the moment I am using vcr but who knows what we'll want to use in the future.

Reply:
I am hoping to use webfakes' OAuth functions for tests at some point. :-)